gulp/jscs.js

```diff
@@ -1,7 +1,11 @@
 var gulp = require('gulp');
+var jspaths = require('./util/jspaths');
 var jscs = require('gulp-jscs');
 
+var fs = require("fs");
+var config = fs.readFileSync("./node_modules/mofo-style/linters/.jscsrc");
```

Comment:
is this a thing that we're okay with being a manual PR realisation, or do we want to somehow automated it as a "this is not wrong, but did you really want a buffer, or did you want a string?"